Fixing typos, uniformisation of help text style, replace PR #2699

imifos opened the pull request:
Replaces https://github.com/ethereum/cpp-ethereum/pull/2699

gavofyork commented on eth/main.cpp:
```diff
@@ -106,74 +108,74 @@ void help()
 		<< "Import/export modes:" << endl
-		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
-		<< "    --to <n>  Export only to block n (inclusive); n may be a decimal, a '0x' prefixed hash, or 'latest'." << endl
+		<< "    --from <n>  Export only from block n; n may be a decimal, a '0x' pre-fixed hash, or 'latest'." << endl
```
"prefixed" (no hyphen) is fine. see [definition of "to prefix" verb](http://www.thefreedictionary.com/prefix).

imifos replied:
Agree, I did that before my first 0xc0ffee :/

gavofyork commented on eth/main.cpp:
```diff
@@ -492,7 +494,7 @@ int main(int argc, char** argv)
 			clientName = argv[++i];
 		}
-		else if ((arg == "-a" || arg == "--address" || arg == "--coinbase-address") && i + 1 < argc)
+		else if ((arg == "-a" || arg == "--address" || arg == "--etherbase-address" || arg == "--coinbase-address") && i + 1 < argc)
```
prefer `--beneficiary` over `--{ether,coin}base-address`.

imifos replied:
Yeah! Totally great (/me hyperbole fan)! That would make it totally clear to users!

gavofyork commented:
Could of minor alterations would be nice, but good otherwise :)

imifos commented:
Corrected the pre-fix typo and removed the coin- to etherbase modification.

Changing the coinbase to for example beneficiary (as discussed), would break compatibility and should be done the correct way, via issue etc.

gavofyork commented:
rebase needed.
